datalad/__init__.py

```diff
@@ -54,6 +53,7 @@ def test(package='datalad', **kwargs):
 
 
 def setup_package():
+    from .log import lgr
```

Comment:
those aren't needed strictly speaking, right? since now there is a `from .log import lgr` above it